README.md

```diff
@@ -52,7 +52,8 @@ echo "LOCKBOX_MASTER_KEY=0000000000000000000000000000000000000000000000000000000
 
 ### Database / Cache
 
-1. Create a database called `covidliste_development` using your favorite postgresql GUI or CLI.
+1. Create a database called `covidliste_development` using your favorite
+   postgresql GUI or CLI.
```

Comment:
We shoud avoid the manual step here anf follow along with the configuration setup in `config/database.yml`. There 's indeed a rake command dedicated to this step: `bin/rails db:create`

Reply:
Agreed. I'll merge this one so we may go on and open a basic PR including your suggestions.